rnd/autogpt_server/autogpt_server/data/block.py

```diff
@@ -1,140 +1,112 @@
-import json
 from abc import ABC, abstractmethod
-from typing import Any, Generator, ClassVar
+from typing import Any, cast, ClassVar, Generator, Generic, TypeVar, Type
 
+import jsonref
 import jsonschema
 from prisma.models import AgentBlock
 from pydantic import BaseModel
 
+from autogpt_server.util import json
+
 BlockData = dict[str, Any]
 
 
 class BlockSchema(BaseModel):
-    """
```

Comment:
I removed manual definition of input_schema and output_schema, now all block will require a pydantic object model to define its schema